package.json

```diff
@@ -36,8 +36,8 @@
     "eslint-plugin-unicorn": "45.0.2",
     "lint-staged": "13.1.0",
     "simple-git-hooks": "2.8.1",
-    "stylelint": "14.16.1",
-    "stylelint-config-standard": "29.0.0"
+    "stylelint": "15.7.0",
```

Comment:
Revert this for appropriate task please

Reply:
OK but this I need this changes for update stylelint-config-standard-scss
I can pull this changes after update root packages will be done.